src/Sentry.Unity/SentryInitialization.cs

```diff
@@ -31,6 +32,12 @@ public static class SentryInitialization
         [RuntimeInitializeOnLoadMethod(RuntimeInitializeLoadType.BeforeSceneLoad)]
         public static void Init()
         {
+            if (!File.Exists($"{Application.dataPath}/Resources/{UnitySentryOptions.ConfigRootFolder}/{UnitySentryOptions.ConfigName}.json"))
```

Comment:
Is this path being built elsewhere? In a way that we can call a function to get it instead of building it in different locations (potentially going out of sync)?